src/common/routes.js

```diff
@@ -39,13 +39,12 @@ export const tokenHandler = (req, res) => {
 /**
  * Handler for the TwiML App Webhook, set in the User's Twilio Console.
  */
-export const twimlHandler = (req, res, options) => {
+export const twimlHandler = (req, res, componentUrl, options) => {
```

Comment:
We can consider doing something like `(req, res, componentUrl, options = {})` and later when we invoke this method and we don't want to "set any options" we can just omit the last parameter. Most languages that allow optional parameters enforce the optional params to be at the end so you can shorthand the invocation like this. Very python-esque. I'm of the opinion that we should always only have one "optional" parameter named "option" like this and then we can destructure it however we like in the method. However, without TS to define the structure of the "option" object I get that it might be a bit obscure at first glance. Anyways, what do you think? Sorry for the ramble lol.